code/game/objects/items/tools/screwdriver.dm

```diff
@@ -32,7 +32,7 @@
 
 /obj/item/screwdriver/New(loc, var/param_color = null)
 	..()
-	if(random_color)
+	if(color_variants && !istype(src, /obj/item/screwdriver/abductor))
```

Comment:
Instead of checking if all screwdrivers are not abductor ones, I suggest replacing this color picking with a proc so you can override it for abductor screwdrivers. For example:
```js
/obj/item/screwdriver/proc/pick_color()
	if(random_color)
		if(!param_color)
			param_color = pick("red","blue","pink","brown","green","cyan","yellow")
		icon_state = "screwdriver_[param_color]"
```
And then for abductor screwdrivers you can do 
```js
/obj/item/screwdriver/abductor/pick_color()
    return
```
You can use this same logic for wirecutters too.

Reply:
Never messed with procs before. *shrug